ambry-network/src/main/java/com.github.ambry.network/NetworkClient.java

```diff
@@ -93,6 +93,7 @@ public NetworkClient(Selector selector, NetworkConfig networkConfig, NetworkMetr
   public List<ResponseInfo> sendAndPoll(List<RequestInfo> requestInfos)
       throws IOException {
     long startTime = time.milliseconds();
+    logger.trace("Initiating Send and poll at " + startTime);
```

Comment:
Lets say we see some cycles of send and poll takes more time thabn avg. So, enabling trace might help us investigate only those cycles which had more latency than expected. May be I can remove the start time. But would like to have trace logging for the time taken for sendAndPoll().